src/components/Modal.tsx

```diff
@@ -36,7 +36,6 @@ export const Modal: React.FC<ModalProps> = ({
   >
     <ModalTitle data-cy="modal-title">{title}</ModalTitle>
     {children}
-    {footer && <Footer>{footer}</Footer>}
```

Comment:
the footer prop was originally rendered as a child of the Modal component instead of the props to prevent adding a line above the footer and to add padding underneath the footer like it is depicted in the designs. can we change it back to how it was before and add a style to Footer on line 51 to handle button spacing?